pkg/ccl/utilccl/licenseccl/license.go

```diff
@@ -72,7 +72,13 @@ func (l *License) Check(at time.Time, cluster uuid.UUID, org, feature string) er
 	// suddenly throwing errors at them.
 	if l.ValidUntilUnixSec > 0 && l.Type != License_Enterprise {
 		if expiration := timeutil.Unix(l.ValidUntilUnixSec, 0); at.After(expiration) {
-			return errors.Errorf("license expired at %s", expiration.String())
+			return errors.Errorf(
```

Comment:
pedantic nit: it might not have been a "trial" license -- while Enterprise licenses aren't checked, the current code means this could also be e.g. a non-commercial license. That said, I'm fine with with the possible slight incorrectness as-is, or I think it reads almost the same without the word "trial" if we wanted to be more strictly accurate.